src/Microsoft.DotNet.Cli.Utils/Command.cs

```diff
@@ -100,7 +102,11 @@ private Command(CommandSpec commandSpec)
                 outputPath: outputPath,
                 applicationName: applicationName);
 
-            if (commandSpec == null)
+            if (_knownCommandsAvailableAsDotNetTool.Contains(commandName, StringComparer.InvariantCultureIgnoreCase) && commandSpec == null)
```

Comment:
Why InvariantCultureIngoreCase? Packages are OrdinalIgnoreCase as is Windows file system. I think it should be OridinalIgnoreCase.

Reply:
Changed to OrdinalIgnoreCase